### ISSUE TITLE
'oa;command' is not working as expected but '/script oa;script' is

### ISSUE THREAD

**Issue by guerman5** [vicalloy/telegram-shell-bot#14]
Hi dear @vicalloy and friends, thanks for the great tool

I've implemented TG-shell-bot in an old android6 with termux, its running awesome with pm2

**The issue:** following the instruction how to use `oa;`  in the case `oa;command`
"add oa in command to send all output. E.g.:"
` oa;ping google.com`
` /script oa;demo.py`

only `/script oa;script` is working 
![image](https://user-images.githubusercontent.com/49020044/148711964-f420139c-0470-48cf-962f-f6bb806f3286.png)

I tried to change the CMD_BLACK_CHARS in settings file, thinking that `';'` might be the culprit, but results were the same
![image](https://user-images.githubusercontent.com/49020044/148711059-c7693b63-be18-4da4-beb8-f71e591e977d.png)

Hope its just a typo somewhere, easy fix.

Happy new year!
Cheers
Guer

**Comment by vicalloy** [comment]
I tested on web and ios client, `oa;ping google.com` is works ok.

**Comment by guerman5** [comment]
Thanks for your time.
I also used ios client to send the command (screenshots above). Notice that my TGshellbot is running in Termux. what about your environment?

**Comment by vicalloy** [comment]
You can add some debug code to see what happend.
ex:
```
@restricted
def do_exec(update, context):
    print('enter do_exec with message: ', update.message)
    if not update.message:
        return
    if __do_cd(update, context):
        print('do cd and return')
        return
    cmd: str = update.message.text
    if not __check_cmd(cmd):
        print('cmd is not allowed:', cmd)
        return
    print('__do_exec')
    __do_exec(cmd, update, context)
```

**Comment by guerman5** [comment]
I added the print lines , after running the different command syntaxes , no extra output on the chat window nor live log entries.
Dont know what I'm doing wrong, I might be confused what I should expect to happen.

I did update the code, cloned the changed files. Same result, same issue on my side
Then I fresh installed in another environment (Archlinux), runs exactly as it does in Termux. 

The only settings i modify are the API_TOKEN, USERS and CMD_WHITE


```
TOKEN = os.environ.get("TELEGRAM_API_TOKEN", '123:sfdDKF_FKDjdf45')
ENABLED_USERS = os.environ.get("ENABLED_USERS", '11111')
CMD_WHITE_LIST = {'date', 'ping', 'ls'}
```

So, if is not the environment and the settings are ok, why is not working :S
